packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,122 @@
+| `first-day-of-week`        | `0, 1, 2, 3, 4, 5, 6`                                   | `0`               | The first day of the week for locale                                                                                              |
+| `first-week-of-year`       | `0, 1, 2`                                               | `0`               | Enum that defines when the first week of the yera should start: first day (`0`), first full week (`1`), first four day week (`2`) |
+| `min-date`                 | string                                                  | -                 | The minimum allowable date                                                                                                        |
+| `max-date`                 | string                                                  | -                 | The maximum allowable date                                                                                                        |
+| `show-link`                | boolean                                                 | `true`            | If false, link at link slot is hidden                                                                                             |
+| `highlight-current-month`  | boolean                                                 | `false`           | Whether the month picker should highlight the current month                                                                       |
+| `highlight-selected-month` | boolean                                                 | `false`           | Whether the month picker should highlight the selected month                                                                      |
+| `today`                    | string                                                  | -                 | value of today                                                                                                                    |
+| `selected-dates`           | string[]                                                | -                 | selected dates                                                                                                                    |
+
+### Outputs
+
+- [selectedDates: string] - the selected dates from the calendar
+
+### Events
+
+- `change`: Emits when the selected date(s) is changed
+- `date-select`: Emits when user selects a date(s) when no date(s) is currently selected
+- `view-change`: Emits when view changes (e.g. from date picker to month picker) via user interaction
+
+### Slots
+
+- `start` - content before the calendar
+- `end` - content after the calendar
+- `link` - link at the bottom right of the base of the calendar (e.g. "Go to today")
+- `navicon-left` - left icon on header to navigate backwards for months, years, or range of years
+- `navicon-right` - right icon on header to navigate forwards for months, years, or range of years
+
```

Comment:
Is there a default slot? If so, what content does it get?

Reply:
Made a note about this in the slots section.